FWCore/Framework/src/Schedule.cc

```diff
@@ -927,8 +940,8 @@ namespace edm {
     
     globalSchedule_->replaceModule(newMod,iLabel);
 
-    for(auto s: streamSchedules_) {
-      s->replaceModule(newMod,iLabel);
+    for(auto& s: streamSchedules_) {
+      get_underlying_safe(s)->replaceModule(newMod,iLabel);
```

Comment:
@Dr15Jones You are correct. get_underlying_safe here will be removed in the next commit.